net/uradvd/files/etc/config/uradvd

```diff
@@ -0,0 +1,13 @@
+
+# simple IPv6 SLAAC advertisement
+config interface
+	# Interface to advertise on
+	option ifname 'br-lan'
+	# default lifetime in seconds (0: forever)
+	option lifetime 65000
+	# prefix on link
+	list prefix_p '300::/64'
+	# prefix not on link
+	#list prefix_a '200::/64'
```

Comment:
How about the following option names?
- `prefix_on_link`
- `prefix_off_link`
- `default_lifetime`
The "default" in that option name is very important - this is the lifetime of the default route!

Reply:
For Gluon is would be nice to have something like `option enabled '0'` in the example config, so the initscript doesn't do anything by default.